### ISSUE TITLE
Stat Issues

### ISSUE THREAD

**Issue by Anthony4444** [GregHilmes/pokebase#8]
There is no easy way to access and individual Pokemon's base stats because a Pokemon class object p has conflicting stats if trying to access as a dictionary. Using p.stats[i]['base_stat'] does not work either for some reason(Error was," TypeError: 'APIMetadata' object is not subscriptable "). 

 

``       
My solution for getting the base_stats:

           i=5
           while i>=0:
            current=str(p.stats[i])
            current=ast.literal_eval(current)
            print (current['stat']['name'])
            print (current['base_stat'])
            i=i-1
``

**Comment by GregHilmes** [comment]
I'm a little confused about what data you're trying to find, but I think I see the issue. **`p.stats[0]` for for the speed stat is a little misleading and confusing.** I plan to change this to `p.stats.speed` to reference the `APIMetadata` object, as soon as I (or someone) gets the time to update the code. 

To give you a temporary solution until this can be fixed, I found Bulbasaur's base speed stat by using the following.
```Python
>>> str(p.stats[0].stat)
'speed'
>>> p.stats[0].base_stat
45
```
... given that `p` is a reference to Bulbasaur's data from the API (`pb.pokemon(1)`). In this example, instead of _subscripting_ `p.stats[0]` (which is what your code is doing), I referenced the _attribute_ `stat` or `base_stat`.
